src/components/Player.vue

```diff
@@ -208,9 +208,11 @@ export default {
     doSeek() {
       if (this.doSeek) {
         this.$store.commit("player/setDoSeek");
-        this.$refs.audio.currentTime = this.seekTime;
-        if (this.playing) {
-          this.$refs.audio.play();
+        if (Number.isFinite(this.seekTime)) {
```

Comment:
Why?

Reply:
This callback gets checked before `onAudioError` is called and kept producing a ton of error messages in the console. This isn't necessary per se.